component-samples/demo/scripts/fdo_toolkit.sh

```diff
@@ -0,0 +1,335 @@
+#!/bin/bash
+
+# This script takes care of the E2E execution of the FDO Service
+
+# All methods will go here
+
+
+
+#################    Documentation   #############################
+#
+#
+#  Usage:
+#
+#  fdo_toolkit.sh
```

Comment:
The script execution will work on Ubuntu-22/Ubuntu-20/Debian, It wont work on RHEL.
can it be updated in the scripts same as well?